package-lock.json

```diff
@@ -1,1346 +1,946 @@
 {
   "name": "learnjs",
   "version": "0.1.0",
-  "lockfileVersion": 3,
+  "lockfileVersion": 1,
   "requires": true,
-  "packages": {
-    "": {
-      "name": "learnjs",
-      "version": "0.1.0",
-      "dependencies": {
-        "@chakra-ui/react": "^2.7.1",
-        "@emotion/react": "^11.11.1",
-        "@emotion/styled": "^11.11.0",
-        "date-fns": "^2.30.0",
-        "framer-motion": "^10.12.17",
-        "lodash": "^4.17.21",
-        "react": "18.2.0",
-        "react-dom": "18.2.0",
-        "react-icons": "^4.10.1",
-        "react-markdown": "^8.0.7",
-        "react-router-dom": "^6.14.1",
-        "react-syntax-highlighter": "^15.5.0",
-        "remark-gfm": "^3.0.1",
-        "vite-tsconfig-paths": "^4.2.0"
-      },
-      "devDependencies": {
-        "@types/react": "^18.0.37",
-        "@types/react-dom": "^18.0.11",
-        "@types/react-syntax-highlighter": "^15.5.7",
-        "@typescript-eslint/eslint-plugin": "^5.59.0",
-        "@typescript-eslint/parser": "^5.59.0",
-        "@vitejs/plugin-react-swc": "^3.0.0",
-        "eslint": "^8.38.0",
-        "eslint-plugin-react-hooks": "^4.6.0",
-        "eslint-plugin-react-refresh": "^0.3.4",
-        "typescript": "^5.0.2",
-        "vite": "^4.3.9"
-      }
-    },
-    "node_modules/@babel/code-frame": {
+  "dependencies": {
+    "@aashutoshrathi/word-wrap": {
+      "version": "1.2.6",
+      "resolved": "https://registry.npmjs.org/@aashutoshrathi/word-wrap/-/word-wrap-1.2.6.tgz",
+      "integrity": "sha512-1Yjs2SvM8TflER/OD3cOjhWWOZb58A2t7wpE2S9XfBYTiIl+XFhQG2bjy4Pu1I+EAlCNUzRDYDdFwFYUKvXcIA==",
+      "dev": true
+    },
+    "@babel/code-frame": {
       "version": "7.22.5",
       "resolved": "https://registry.npmjs.org/@babel/code-frame/-/code-frame-7.22.5.tgz",
       "integrity": "sha512-Xmwn266vad+6DAqEB2A6V/CcZVp62BbwVmcOJc2RPuwih1kw02TjQvWVWlcKGbBPd+8/0V5DEkOcizRGYsspYQ==",
-      "dependencies": {
+      "requires": {
         "@babel/highlight": "^7.22.5"
-      },
-      "engines": {
-        "node": ">=6.9.0"
       }
     },
-    "node_modules/@babel/helper-module-imports": {
+    "@babel/helper-module-imports": {
```

Comment:
Suggest to not commit these `package-lock.json` changes.

It appears that `node_modules/` is being removed from each of these packages.  Not sure if this is contributing to the build failure, but the error messages does seem to reference `node_modules`?

<details>
<summary>Click me to see screenshots</summary>

![Screenshot 2023-07-29 at 9 00 43 AM](https://github.com/mitxpro-dev/learnjs/assets/96446064/2aa3a1e4-d72e-45cf-b9a0-f1a7102e6fdb)




![Screenshot 2023-07-29 at 9 02 00 AM](https://github.com/mitxpro-dev/learnjs/assets/96446064/da631420-2fa5-4709-a29c-5295ead30dd1)




  
</details>